src/errors.ts

```diff
@@ -84,6 +84,11 @@ class ErrorPolykeyCLIUnexpectedError<T> extends ErrorPolykeyCLI<T> {
   exitCode = sysexits.SOFTWARE;
 }
 
+class ErrorPolykeyCLISubprocessFailure<T> extends ErrorPolykeyCLI<T> {
+  static description = 'A subprocess failed to exit gracefully';
+  exitCode = sysexits.UNKNOWN;
+}
+
```

Comment:
Can you change the description and name according to https://github.com/MatrixAI/Polykey-CLI/pull/329#issuecomment-2481400933

Reply:
I had actually merged this PR and had forgotten to make this change. So, I made this change in another PR I was working on. This error has been renamed to `ErrorPolykeyCLIChildProcessFailure` with the updated description in https://github.com/MatrixAI/Polykey-CLI/pull/336.

Reply:
Is this not specific to the env command? If it's called `ErrorPolykeyCLIChildProcessFailure` that would imply in general the idea of a child process failure to PK CLI.

Unless you're also using this in any situation you're going to create a child process?

Reply:
This error is not specific to the `env` command. As `secrets edit` also spawns the editor in a child process, any failure of the child process is wrapped in this error.

So, yes, in any situation where a child process might fail, this error should be used to wrap it, unless there is a better error in that context.